docs/integrations/unit-testing.md

```diff
@@ -225,6 +225,15 @@ def test_function():
 
 If the function raises a ValueError with proper error message, the test will pass.
 
+## Content Testing Guidelines
+
```

Comment:
would add adise to mock demistomock directly using mocker, not rely on what we have in the file